docs/tutorials/simple-todo-list/3-the-todo-model.md

```diff
@@ -87,7 +88,7 @@ Your database (`db.sqlite3`) now contains a new table named `todo`:
 |                             todo                             |
 +------------+-----------+-------------------------------------+
 | id         | integer   | PRIMARY KEY AUTO_INCREMENT NOT NULL |
-| text       | varchar   |                                     |
+| text       | varchar   | NOT NULL                            |
```

Comment:
Does the `text` column really accept `NULL` values? It looks like from the migration file and the entity class (primitive types are by default not nullable in TypeORM) that the `text` field cannot be null.

Reply:
You're right, not nullable.

I guess explicit NOT NULL in this table is alright for clarity.

Reply:
Sorry I got it wrong. I thought that this PR was removing the `NOT NULL` constraint whereas it's the opposite.